packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -881,6 +881,8 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   /// Typically this would be set to [CupertinoColors.inactiveGray].
   ///
   /// If this is null, the background cursor is not painted.
+  ///
+  /// @{macro flutter.services.TextInput.floatingCursor}
```

Comment:
i would just inline the see also in all these places. using a macro means people browsing the source can't easily see what you're trying to inline, and if you ever want to use the macros somewhere that already has a "see also" you'll need to find another solution.